README.md

```diff
@@ -234,6 +234,23 @@ If you aren’t using React, you can still render a player using the standalone
 
 See [`jsFiddle` example](https://jsfiddle.net/krkcvx9s/)
 
+#### Adding your own players
+
+If you have your own player, that compatible with ReactPlayer's internal architecture, you can use it like this:
+
+```javascript
+import YourOwnPlayer from './somewhere';
+ReactPlayer.addAdditionalPlayer(YourOwnPlayer);
+```
+
+Or you can clear all additional players:
+
+```javascript
+ReactPlayer.clearAdditionalPlayers();
+```
+
+All responsibilities for keeping your player's API compatible with ReactPlayer is on you.
```

Comment:
Not sure this line is necessary? Seems a bit aggressive.

Reply:
@CookPete i just wanted to protect you from issues related to changes in interfaces of your "internal" players.